docs/network/ble/ble_intro.md

```diff
@@ -51,5 +60,6 @@ Mynewt comes with two built-in projects that allow users to play with NimBLE, tr
 
 1. **bletiny** : A simple shell application which provides a basic interface to the host-side of the BLE stack. 
 2. **bleprph**: A basic peripheral device with no user interface. It advertises automatically on startup, and resumes advertising whenever a connection is terminated. It supports a maximum of one connection.
+3. **blemesh**: Sample application for BLE Mesh node with on/off model.
```

Comment:
Change "Sample application" -> "A sample application" to make the list consistent. 
I am not familiar with BLE Mesh nodes. The literature indicates a server/client model. Does this application implement a server, client, or both?

Reply:
"Node" defines that.